src/core/symbology/qgspointdisplacementrenderer.h

```diff
@@ -99,6 +99,19 @@ class CORE_EXPORT QgsPointDisplacementRenderer: public QgsPointDistanceRenderer
      */
     double circleRadiusAddition() const { return mCircleRadiusAddition; }
 
+    /**
+     * Sets a factor for increasing the label distances from the symbol.
+     * \param factor addition factor
+     * \see labelDistanceFactor()
+     */
```

Comment:
\since QGIS 3.8

Reply:
I didn't see this comment before, but it's added now.